frontend/src/components/CytoscapeGraph/ContextMenu/EdgeContextMenu.tsx

```diff
@@ -12,14 +12,12 @@ const contextMenu = kialiStyle({
   textAlign: 'left'
 });
 
-export class EdgeContextMenu extends React.PureComponent<EdgeContextMenuProps> {
-  render() {
-    return (
-      <div className={contextMenu}>
-        {getTitle(`Edge (${prettyProtocol(this.props.protocol)})`)}
-        {renderBadgedName(decoratedNodeData((this.props.element as EdgeSingular).source()), 'From:  ')}
-        {renderBadgedName(decoratedNodeData((this.props.element as EdgeSingular).target()), 'To:        ')}
-      </div>
-    );
-  }
-}
+export const EdgeContextMenu: React.FC<EdgeContextMenuProps> = (props: EdgeContextMenuProps) => {
```

Comment:
I'm fairly sure Edge context menus never worked or aren't used.  But this if fine.